packages/jsdoc-plugins/lib/doclet.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Doclets are atomic parts of the JSDoc output that describe each symbol of parsed input.
+ *
+ * @typedef {Object} Doclet
```

Comment:
In order to increase the readability, I would like to add a blank line before `@property` tag.